packages/dialog/stories/styled-dialog-content.js

```diff
@@ -0,0 +1,38 @@
+import { LitElement, html, css } from '@lion/core';
+
+class StyledDialogContent extends LitElement {
+  static get styles() {
+    return [
+      css`
+        :host {
+          background-color: #fff;
+        }
+        .nice {
+          font-weight: bold;
+          color: green;
+        }
+        .close-button {
+          color: black;
+          font-size: 28px;
+          line-height: 28px;
+        }
+      `,
+    ];
+  }
+
+  _closeOverlay() {
+    this.dispatchEvent(new Event('close-overlay', { bubbles: true }));
```

Comment:
I think this is a great way of solving the issue you had, which is that the close-overlay does not cross the shadow boundary of your styled component, which means you cannot close your dialog. So then you make it a method on your styled component and (re)dispatch the close-overlay event from the component rather than the <button> in your template and putting `composed: true` which I would consider the "bad" solution. 

I think this by itself deserves documentation as well :) because otherwise lots of people would do this incorrectly I think.

Reply:
I added this documentation in a new commit. I'm using the same example though.